data/fixtures/scopes/zig/name.argument.formal.scope

```diff
@@ -0,0 +1,55 @@
+fn foo(aaa: u8, bbb: u8) void {}
+---
+
+[#1 Content] =
+[#1 Removal] = 0:3-0:6
+     >---<
+0| fn foo(aaa: u8, bbb: u8) void {}
+
+[#1 Leading delimiter] = 0:2-0:3
+    >-<
+0| fn foo(aaa: u8, bbb: u8) void {}
+
+[#1 Domain] = 0:0-0:32
+  >--------------------------------<
+0| fn foo(aaa: u8, bbb: u8) void {}
+
+[#1 Insertion delimiter] = " "
+
+
+[#2 Content] = 0:7-0:10
+         >---<
+0| fn foo(aaa: u8, bbb: u8) void {}
+
+[#2 Removal] = 0:7-0:16
+         >---------<
+0| fn foo(aaa: u8, bbb: u8) void {}
```

Comment:
In other languages we don't include the type in the rim wall range. Please just make the removable range the same as the content range.